### PR TITLE
Remove repetetive sense in CMFD that cancels out

### PR DESCRIPTION
**Pull request by loganharbour**
After working with @GiudGiud for a bit, we came to the conclusion that this term cancels out.

Hopefully the test suite will agree?﻿

### CODE REVIEW

**Comment by GiudGiud** [comment]
Looks like CI suite is down. I ll just check locally

File: src/Cmfd.cpp [hunk 1]
```diff
@@ -1140,23 +1138,23 @@ void Cmfd::getSurfaceDiffusionCoefficient(int cmfd_cell, int surface,
...
-    dif_surf_corr = -(sense * dif_surf * (flux_next - flux) + current)
+    dif_surf_corr = -(dif_surf * (flux_next - flux) + current)
         / (flux_next + flux);
 
     /* Flux limiting condition */
     if (_flux_limiting && _moc_iteration > 0) {
       double ratio = dif_surf_corr / dif_surf;
       if (std::abs(ratio) > 1.0) {
 
-        if (sense * current > 0.0)
```
Comment (GiudGiud) [inline src/Cmfd.cpp:1154]: well there s the reason sense is here
This seems to be a test to see if it's outgoing or not.

dont think we can change that line.

**Review by GiudGiud — CHANGES_REQUESTED** [review]

File: src/Cmfd.cpp (hunk 1 above)
Comment (loganharbour) [reply src/Cmfd.cpp:1154]: Oh duh - yeah that makes sense. I'll add back.

File: src/Cmfd.cpp [hunk 2]
```diff
@@ -1458,12 +1456,12 @@ void Cmfd::constructMatrices() {
...
-          value = (dif_surf - sense * dif_surf_corr) * delta;
+          value = (dif_surf - dif_surf_corr) * delta;
           _A->incrementValue(i, e, i, e, value);
 
           /* Set the off diagonal term */
           if (getCellNext(i, s, false, false) != -1) {
-            value = - (dif_surf + sense * dif_surf_corr) * delta;
+            value = - (dif_surf + dif_surf_corr) * delta;
             _A->incrementValue(getCellNext(i, s, false, false), e, i, e, value);
           }
```
Comment (GiudGiud) [inline src/Cmfd.cpp:1466]: there's another sense to remove ten lines below, it s for the off-domain term

File: src/Cmfd.cpp (hunk 1 above)
Comment (GiudGiud) [reply src/Cmfd.cpp:1154]: nah you're right it cancels out here too.

**Comment by loganharbour** [comment]
@GiudGiud should be good to go

**Review by GiudGiud — APPROVED** [review]

**Pull request merged**